.gitignore

```diff
@@ -38,8 +38,9 @@ logs
 
 # Eclipse specific ignore settings
 .settings/
-.project
-.pmdruleset.xml
-.pmd
-.classpath
-application.properties
+*.properties
+/dashboard.properties
+
+/appdynamics-collector/src/main/resources/appdynamics.properties
+/appdynamics-collector/appdynamics.properties
+/api/api.properties
```

Comment:
The appdynamics specfic stuff should be within the appdynamics collector .gitignore.